app/code/community/Aoe/JsCssTstamp/Block/Head.php

```diff
@@ -264,39 +264,56 @@ protected function _separateOtherHtmlHeadElements(&$lines, $itemIf, $itemType, $
      */
     protected function &_prepareStaticAndSkinElements($format, array $staticItems, array $skinItems, $mergeCallback = null)
     {
+        $versionKey = Mage::getModel('aoejscsststamp/package')->getVersionKey();
+        $AddTstampToAssetsJs = Mage::getModel('aoejscsststamp/package')->getAddTstampToAssetsJs();
```

Comment:
This call does not work. I would recommend to change it to:
$AddTstampToAssetsJs = Mage::getStoreConfig('dev/js/addTstampToJsFiles');